tasks.py

```diff
@@ -24,7 +24,7 @@
 
 @functools.lru_cache(maxsize=8)
 def _get_pyproject_tool_dict(
-    tool_key: Literal["poetry", "black", "ruff", "mypy"] | None = None
```

Comment:
Without this change the action to run `poetry run invoke fmt --check` was failing